config/test/kubemacpool.yaml

```diff
@@ -260,7 +260,7 @@ spec:
       containers:
       - args:
         - --v=debug
-        - --wait-time=10
+        - --wait-time=600
```

Comment:
I think we should leave config/test w/o wait-time argument and change the default value in main.go to be 60
so we could iterate faster on test suite.

Reply:
We need to match test & production behavior, to eliminate hidden issues like the one this commit has exposed.